packages/extension/test/components/dashboard/DashboardBottom.test.js

```diff
@@ -20,13 +21,122 @@ const getComponent = (props = {}) => {
   )
 }
 
+
+
+const props = {
```

Comment:
If this object is only going to be used in 2 tests, maybe this should either be in those 2 tests, or called something "defaultProps"?

Reply:
Done